### DISCUSSION TITLE
What should I use for RESOURCE_TOKEN

### DISCUSSION THREAD

**Discussion by roalexan** [Azure-Samples/chat-with-your-data-solution-accelerator#895] [General]
The usage doc says: "https://web-{RESOURCE_TOKEN}-admin.azurewebsites.net/

Where {RESOURCE_TOKEN} is uniquely generated during deployment. This is a combination of your subscription and the name of the resource group. Then select Ingest Data and add your data. You can find sample data in the /data directory"

I'm not clear what that means. Where can I get the value to use? Is it an output somewhere during the deployment?

**Comment by pmariani** [comment]
Once deployment is complete you can go to your resources and open the admin app service for example, and its URL will be output on the overview page. It will have the random resource token.

**Comment by ross-p-smith** [comment]
Hi @roalexan - if you deploy using azd; then you will be asked for the name of your `AZURE_ENV_NAME` - this could be `123` as an example. This will used to create a Resource Group called `rg_123`. We will then take your subscription id and `AZURE_ENV_NAME` and create a unique resource token. This is because some of the resources have to be globally unique.

If you deploy through the ARM template (Deploy to Azure button) - then it will be similar
![image](https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/assets/6254153/89a58e43-05a8-416e-b679-a52918accba9)

If you use azd to deploy you will find 2 environment variables in the `.env` file. `ADMIN_WEBSITE_NAME` and `FRONTEND_WEBSITE_NAME`
